LongWarOfTheChosen/Src/LW_PerkPack_Integrated/Classes/X2Effect_SteadyWeapon.uc

```diff
@@ -105,18 +110,49 @@ static function EventListenerReturn SteadyWeaponWoundListener(Object EventData,
 function GetToHitModifiers(XComGameState_Effect EffectState, XComGameState_Unit Attacker, XComGameState_Unit Target, XComGameState_Ability AbilityState, class<X2AbilityToHitCalc> ToHitType, bool bMelee, bool bFlanking, bool bIndirectFire, out array<ShotModifierInfo> ShotModifiers)
 {
 	local ShotModifierInfo ShotInfo;
+	local XComGameState_Item SourceWeaponState;
+
+	SourceWeaponState = AbilityState.GetSourceWeapon();
 
 	if(!bMelee && AbilityState.SourceWeapon == EffectState.ApplyEffectParameters.ItemStateObjectRef)
```

Comment:
I would put a space after `if` here too, along with the new `if` statements below.